client/lib/cart-values/cart-items.ts

```diff
@@ -868,10 +864,6 @@ export function getDomainPriceRule(
 		return 'PRICE';
 	}
 
-	if ( isDomainOnly ) {
-		return 'PRICE';
-	}
-
```

Comment:
This is the update. The spacing changes above are due to Prettier doing its thing.